HISTORY.rst

```diff
@@ -6,6 +6,36 @@ Release History
 unreleased
 +++++++++++++++
 
+**IoT Central updates**
+
+* `--api-version` parameter will be deprecated and ignored. The IoT Central API will alway call latest GA version or latest preview version (if any API only exists in preview).
```

Comment:
This may be a breaking change.

Reply:
It could be. But customers won't feel any differences since all central APIs are backward compatible, except the oldest preview version. And Johnson has confirmed that no one is using that preview version on CLI. This change just reduce a lot dev efforts later.